EdvClient.js

```diff
@@ -352,29 +352,50 @@ export class EdvClient {
    *   capability to use to authorize the invocation of this operation.
    * @param {Object} options.invocationSigner - An API with an
    *   `id` property and a `sign` function for signing a capability invocation.
+   * @param {function} [keyResolver=this.keyResolver] a function that returns
+   *   a Promise that resolves a key ID to a DH public key.
    *
    * @return {Promise<Boolean>} resolves to `true` if the document was deleted
    *   and `false` if it did not exist.
    */
-  async delete({id, capability, invocationSigner}) {
+  async delete({id, capability, invocationSigner,
+    keyResolver = this.keyResolver}) {
```

Comment:
Whenever we need to move a function parameter down a line to accommodate the 80 character limit, we move all parameters down to visually break them up from the function itself. I don't think we have a linting rule that can catch that at the moment, unfortunately.

Not sure if this suggestion meets the 80 char threshold ... but this would be the first suggestion:

```suggestion
  async delete({
    id, capability, invocationSigner, keyResolver = this.keyResolver
  } = {}) {
```

If that's still over:

```suggestion
  async delete({
    id, capability, invocationSigner,
    keyResolver = this.keyResolver
  } = {}) {
```